docs/src/ddns.md

```diff
@@ -0,0 +1,58 @@
+# Dynamic Decision Networks
+
+Part of the conceptual definition of a POMDP or MDP is a dynamic decision network (DDN) that defines which random variables are dependent on each other.
+Usually, problem writers will not have to interact directly with the DDN, but it is a helpful concept for understanding, and it can be customized for special problem types.
+
+The standard POMDPs.jl DDN models are shown below:
+
+| Standard MDP DDN | Standard POMDP DDN |
+|:---:|:---:|
+|![MDP DDN](figures/mdp_ddn.svg) | ![POMDP DDN](figures/pomdp_ddn.svg) |
+
+!!! note
+
+    These DDNs are slightly different from the ones traditionally used in the POMDP literature in that they have `:sp`→`:r` and `:o`→`:r` edges. Traditional POMDP algorithms will work with this DDN if ``R(s,a)`` is the expectation of ``R(s, a, s', o)`` over all ``s'`` and ``o``.
```

Comment:
IMHO this sounds unnecessarily intimidating. Maybe we can add a sentence like  "These edges were added to provide additional flexibility to problem writers.". It may be obvious/redundant but we don't want anyone to be confused by the fact that we use a different DDN. I'd be a bit concerned that people think "Oh, I can I not use this for the vanilla POMDP?" when only quickly scanning this note. But I may be overly cautious and no-one would think that anyway.

Reply:
good call - I changed it